WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -67,6 +67,16 @@ def notifications_for_user(user)
       }
     end
 
+    user.delegated_competitions.visible.over
+      .where("start_date >= ?", DelegateReport::REPORTS_ENABLED_DATE) # Don't notify about very old competitions without reports.
```

Comment:
I'm supportive of *not* filtering out old competitions. It will motivate delegates to submit reports for their old competitions, and if it really annoys them, they can always submit empty reports.